tests/openvino/test_modeling.py

```diff
@@ -1718,7 +1720,7 @@ def test_generate_utils(self, model_arch):
         gc.collect()
 
```

Comment:
Could we also add a test to make sure that the architectures that supports it are indeed exported in stateful mode ? (so for now for t5, mt5 and whisper)

Reply:
thanks!